cdci_data_analysis/analysis/parameters.py

```diff
@@ -794,7 +818,13 @@ class TimeDelta(Time):
     owl_uris = ("http://odahub.io/ontology#TimeDeltaIsDeprecated",) 
     format_kw = 'delta_T_format'
     
-    def __init__(self, value=None, delta_T_format='sec', name=None, delta_T_format_name=None, par_default_format='sec', extra_metadata=None):
+    def __init__(self, 
+                 value, 
+                 delta_T_format='sec', 
+                 name=None, 
+                 delta_T_format_name=None, 
+                 par_default_format='sec', 
+                 extra_metadata=None):
```

Comment:
is the `is_optional` arg to be added here?

Reply:
I didn't add it because this parameter class is deprecated. It's there to keep compatibility with plugins, but isn't used in nb2workflow plugin, so will never be declared optional.